src/test/java/com/solr/clientwrapper/rest/SolrSchemaResourceTest.java

```diff
@@ -30,12 +31,12 @@
 class SolrSchemaResourceTest {
 
 	String solrendpoint = "/schema";
-	String tableName = "gettingstarted1";
+	String tableName = "gettingstarted2";
 	String name = "default-config";
 	SolrFieldDTO solr = new SolrFieldDTO("testField6", SolrFieldType._nest_path_, "mydefault", true, true, false, true, true);
 	SolrFieldDTO[] attributes = { solr };
 	String expectedGetResponse = "{\n"
-			  +"\"tableName\": \"gettingstarted1\",\n"
+			  +"\"tableName\": \"gettingstarted2\",\n"
```

Comment:
We should take _tableName_ here from the variable declared above, and that too should be coming from configuration file. Try doing that please.